.vscode/lit.code-snippets

```diff
@@ -1,30 +1,36 @@
 {
-	"Create Lit Component": {
-		"prefix": "lit new",
+	"UmbNewLitElement": {
+		"prefix": "new umb element",
 		"scope": "typescript",
 		"body": [
-			"import { UmbTextStyles } from '@umbraco-ui/backoffice/style';",
-			"import { css, html } from 'lit';",
-			"import { customElement } from 'lit/decorators.js';",
+			"import { css, customElement, html } from '@umbraco-cms/backoffice/external/lit';",
 			"import { UmbLitElement } from '@umbraco-cms/backoffice/lit-element';",
+			"import { UmbTextStyles } from '@umbraco-cms/backoffice/style';",
 			"",
-			"@customElement('umb-${TM_FILENAME_BASE/(.*)\\..+$/$1/}')",
-			"export class Umb${TM_FILENAME_BASE/(.*)\\..+$/${1:/pascalcase}/}Element extends UmbLitElement {",
-			"\trender() {",
-			"\t\treturn html`${0:umb-${TM_FILENAME_BASE/(.*)\\..+$/$1/}}`;",
+			"const elementName = 'umb-${TM_FILENAME_BASE/(.*)\\..+$/$1/}';",
+			"",
+			"@customElement(elementName)",
+			"export class Umb${TM_FILENAME_BASE/(.*)$/${1:/pascalcase}/}Element extends UmbLitElement {",
+			"\tconstructor() {",
+			"\t\tsuper();",
+			"\t}",
+			"",
+			"\toverride render() {",
+			"\t\treturn html`$0`;",
 			"\t}",
 			"",
-			"\tstatic styles = [UmbTextStyles, css``];",
+			"\tstatic override styles = [UmbTextStyles, css``];",
```

Comment:
This could be `readonly` as it never changes (and it prevent a Sonarcloud issue):

```suggestion
			"\tstatic override readonly styles = [UmbTextStyles, css``];",
```